district_data_utils.py

```diff
@@ -28,7 +28,7 @@ def district_school_to_dp_school(name):
 
 def dp_grade_for_district_record(district_record):
     # District data uses grade 0 for both TK and Kindergarten
-    if district_record['entrycode'] == 'TK':
+    if district_record['entrycode'] == 'TK' or district_record['Grade'] == 'TK':
         return "-1"
     else:
         return district_record['Grade']
```

Comment:
Contents of the method can be changed to:
```python
return "-1" if district_record['Grade'] == 'TK' else district_record['Grade']
```

Reply:
You mean 
```python
return "-1" if district_record['Grade'] == 'TK' or  if district_record['entrycode'] == 'TK' else district_record['Grade']
```

Reply:
That's not what I mean. With the old exports, BSD listed TK as `Grade` 0 plus `entrycode` 'TK'. Now they just list `Grade` as 'TK'. So, we no longer need to look at `entrycode` to determine grade.

Reply:
I'd rather leave this one as-is, with the check on entry-code. The district data has been really unpredictable, and come the next upload, we will not remember the expected contract, and they may switch back. This catches both cases. Separately, we will need document the expected contract and find a way to verify that the data conforms.

Reply:
It's not going to switch back, it changed because they switched backend systems. I'd rather keep the code as simple as possible.